koku/api/report/test/azure/tests_azure_query_handler.py

```diff
@@ -51,13 +50,13 @@ def setUp(self):
         self.generator = AzureReportDataGenerator(self.tenant)
         self.generator.add_data_to_tenant()
 
-    def get_totals_by_time_scope(self, aggregates, filter=None):
+    def get_totals_by_time_scope(self, aggregates, filters=None):
```

Comment:
This argument rename is to make the linter happy, so that we're not overriding the built-in `filter` keyword.